recipe/meta.yaml

```diff
@@ -31,6 +32,7 @@ requirements:
     - tk
   run:
     - python
+    - vs2015_runtime 14.00.23026.0 0   # [win and py35]
```

Comment:
Yes, I only merged as I wanted us to build as many combinations of Pillow as possible. It definitely should be removed.